### PR TITLE
docs: add websocket section to tutorial

### PR DESCRIPTION
**Pull request by ptgamr**
Add docs for #542 

r: @n1k0

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
 
 We're using [`#resolve()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-resolve) to mark a conflict as resolved: it accepts a conflict object, and a resolution one; the latter is what will be updated locally and sent for resynchronization on a next call to [`#sync()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-sync) — which is exactly what we're doing after we've resolved all our conflicts.
 
+## Receiving changes via Websocket
```
Comment (n1k0) [inline docs/tutorial.md]: I suggest we move this to a new dedicated section of the docs, eg. `Advanced usage`. The tutorial is really aimed at beginners with simple, general use cases.

File: docs/tutorial.md [hunk 2]
```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
...
+
+`collection.sync()` only sync the data after user triggerred the sync. Very often we want to live sync user data among different clients. In that case, you'll need to listen to the changes via a websocket server.
```
Comment (n1k0) [inline docs/tutorial.md]: While explicitly calling `collection.sync()` works nicely, sometimes we want the synchronization operation to happen automatically in the background. In that case, we need to listen to change events sent from a [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) server.

File: docs/tutorial.md [hunk 3]
```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
...
+
+Currently, the simplest way to pushlish the changes via websocket from Kinto API is to integrate with Pusher.com ([tutorial here](https://kinto.readthedocs.io/en/stable/tutorials/notifications-websockets.html)).
```
Comment (n1k0) [inline docs/tutorial.md]: *publish*

File: docs/tutorial.md [hunk 4]
```diff
@@ -495,6 +495,44 @@ Your take really. Let's take the former approach:
...
+Later, [kinto-webpush](https://github.com/Kinto/kinto-webpush/) will help you to use your own websocket server.
+
+When there are changes comming, we can import the changes to local store using [`collection.importChanges()`](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-importChanges):
```
Comment (n1k0) [inline docs/tutorial.md]: - *incoming*
- *we can import them*

**Comment by Natim** [comment]
Fixed in https://github.com/Kinto/kinto.js/commit/df8221c82965ea840b73ca1f2f9ed9da02888dd3

**Pull request closed without merging**